<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Librepad</class>
 <widget class="QMainWindow" name="Librepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Librepad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent">
     <property name="title">
      <string>&amp;Recent Files</string>
     </property>
     <property name="icon">
      <iconset resource="../../../res/icons/icons.qrc">
       <normaloff>:/icons/open.svg</normaloff>:/icons/open.svg</iconset>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_Recent"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionReload"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>&amp;Search</string>
    </property>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Se&amp;ttings</string>
    </property>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionLoadScript"/>
   </widget>
   <widget class="QMenu" name="menu_Widgets">
    <property name="title">
     <string>&amp;Widgets</string>
    </property>
    <widget class="QMenu" name="menu_Toolbars">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>&amp;Toolbars</string>
     </property>
     <addaction name="actionToolBarMain"/>
     <addaction name="actionToolBarSearch"/>
     <addaction name="actionToolBarBuild"/>
    </widget>
    <addaction name="menu_Toolbars"/>
    <addaction name="separator"/>
    <addaction name="actionCmdDock"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSearch"/>
   <addaction name="menuSettings"/>
   <addaction name="menuBuild"/>
   <addaction name="menu_Widgets"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>16777215</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="buildToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="actionLoadScript"/>
  </widget>
  <widget class="QToolBar" name="searchToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>356</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/new.svg</normaloff>:/icons/new.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New text document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/open.svg</normaloff>:/icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/save_as.svg</normaloff>:/icons/save_as.svg</iconset>
   </property>
   <property name="text">
    <string>Save &amp;as</string>
   </property>
   <property name="toolTip">
    <string>Save file as</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/print.svg</normaloff>:/icons/print.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="toolTip">
    <string>Print file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/quit.svg</normaloff>:/icons/quit.svg</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit librepad</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/copy.svg</normaloff>:/icons/copy.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/cut.svg</normaloff>:/icons/cut.svg</iconset>
   </property>
   <property name="text">
    <string>C&amp;ut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/paste.svg</normaloff>:/icons/paste.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/undo.svg</normaloff>:/icons/undo.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/redo.svg</normaloff>:/icons/redo.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/info.svg</normaloff>:/icons/info.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Info</string>
   </property>
   <property name="toolTip">
    <string>About Librepad</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/search_previous.svg</normaloff>:/icons/search_previous.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;previous search result&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/text.svg</normaloff>:/icons/text.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Font</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/search_next.svg</normaloff>:/icons/search_next.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;next search result&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/run.svg</normaloff>:/icons/run.svg</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>run script</string>
   </property>
  </action>
  <action name="actionLoadScript">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/open_script.svg</normaloff>:/icons/open_script.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Script</string>
   </property>
   <property name="toolTip">
    <string>load a script from file</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/reload.svg</normaloff>:/icons/reload.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCmdDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Cmd</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionToolBarMain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;File</string>
   </property>
  </action>
  <action name="actionToolBarSearch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Search</string>
   </property>
  </action>
  <action name="actionToolBarBuild">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Build</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../../../res/icons/icons.qrc">
     <normaloff>:/icons/help.svg</normaloff>:/icons/help.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../res/icons/icons.qrc"/>
  <include location="../../../res/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
